### ISSUE TITLE
Improve Hall_parameter signature and docstring

### ISSUE THREAD

**Issue by rocco8773** [PlasmaPy/PlasmaPy#915]
The current signature to the Hall parameter is...

```python
def Hall_parameter(
    n: u.m ** -3,
    T: u.K,
    B: u.T,
    ion: Particle,
    particle: Particle,
    coulomb_log=None,
    V=None,
    coulomb_log_method="classical",
):
```

It is unclear what arguments `ion` and `particle` refer too and the docstrings do not do any help.  What should be done...

1.  Rename `ion` and `particle` to make them more intuitive.
1. Update docstrings to better describe what `ion` and `particle` are.
1. Add an equation to the docstring.

**Comment by Dhawal-Modi** [comment]
hello i would like to work on this issue

**Comment by rocco8773** [comment]
Have at it!  We did make some improvement to the parameter descriptions, https://github.com/PlasmaPy/PlasmaPy/pull/911#discussion_r504149369, but there's still room for improvement.

**Comment by Dhawal-Modi** [comment]
Alright thanks! i'll see what more i can do.

**Comment by Dhawal-Modi** [comment]
what should i put in <TYPE> for the changelog entry file 915.<TYPE>.rst? Should it be doc or trivial?

**Comment by rocco8773** [comment]
I'd go with `doc`.

**Comment by Dhawal-Modi** [comment]
I have added a pull request for this issue, please review pull request #934 .

**Comment by rocco8773** [comment]
closed by #934
